mitodl/open-discussions
# Added a setting for the JWT cookie name

#### What are the relevant tickets?
N/A

#### What's this PR do?
Adds a setting for the JWT cookie name

#### How should this be manually tested?
Set `OPEN_DISCUSSIONS_COOKIE_NAME=boop` in both MM and OD `.env`
Verify you can still hit /discussions/ on MM, login to OD, and that you see the cookie with the new name set.
